frontend/sw360-portlet/src/main/webapp/html/moderation/view.jsp

```diff
@@ -57,35 +57,37 @@
                 <div id="tab-Open" class="tab-pane">
                     <table id="moderationsTable" cellpadding="0" cellspacing="0" border="0" class="display" style="width:100%">
                         <colgroup>
-                            <col style="width: 5%;" />
+                            <col style="width: 90px;" />
```

Comment:
mixing `px` and `%` is normally not a good idea

Reply:
How does it behave when the window is scaled?

Reply:
yes mixing of px and % is a bad idea.
Anyway I don't like to specify each column width.
I removed this code, thus the browser calculates the optimal column width.
I looks much better, especially the date and type columns are now calculates in one line.